be/src/storage/del_vector.cpp

```diff
@@ -120,4 +120,12 @@ void DelVector::_update_stats() {
     }
 }
 
+void DelVector::copy_from(const DelVector& delvec) {
```

Comment:
Why delete the copy assignment operator and add this method? Is any problem to make a custom copy constructor and copy assignment?

Reply:
I think add `copy_from` function is more clear that `_roaring` have been copy, not move, so I add it.